cibuildwheel/resources/testing_temp_dir_file.py

```diff
@@ -0,0 +1,14 @@
+# this file is copied to the testing cwd, to raise the below error message if
+# pytest is run from there
+
+
+def test():
```

Comment:
This assumes pytest. Would it make more sense to write it in unittest, which pytest will also pick up and run? I'm assuming this still won't get other [testing frameworks](https://github.com/cleder/awesome-python-testing#testing-frameworks) (like [ward](https://github.com/darrenburns/ward)), but this might be better than just assuming pytest?

Reply:
good idea! I'll try that